storagenode/storagenodedb/database.go

```diff
@@ -115,108 +117,83 @@ func New(log *zap.Logger, config Config) (*DB, error) {
 		return nil, err
 	}
 
-	versionsPath := config.Info2
-	versionsDB, err := openDatabase(versionsPath)
-	if err != nil {
-		return nil, err
-	}
-
 	db := &DB{
 		log:    log,
 		pieces: pieces,
 		kdb:    dbs[0],
 		ndb:    dbs[1],
 		adb:    dbs[2],
 
-		// Initialize databases. Currently shares one info.db database file but
-		// in the future these will initialize their own database connections.
-		versionsDB:        newVersionsDB(versionsDB, versionsPath),
-		v0PieceInfoDB:     newV0PieceInfoDB(versionsDB, versionsPath),
-		bandwidthDB:       newBandwidthDB(versionsDB, versionsPath),
-		ordersDB:          newOrdersDB(versionsDB, versionsPath),
-		pieceExpirationDB: newPieceExpirationDB(versionsDB, versionsPath),
-		pieceSpaceUsedDB:  newPieceSpaceUsedDB(versionsDB, versionsPath),
-		reputationDB:      newReputationDB(versionsDB, versionsPath),
-		storageUsageDB:    newStorageusageDB(versionsDB, versionsPath),
-		usedSerialsDB:     newUsedSerialsDB(versionsDB, versionsPath),
-		satellitesDB:      newSatellitesDB(versionsDB, versionsPath),
+		dbDirectory: filepath.Dir(config.Info2),
```

Comment:
I'm wondering whether we should have each database separately configurable or not. There have been at least requests to move some databases to a different drive compared to the blob storage. Or at least let's make a flag that specifies the directory rather than reusing info2.

Also why pass in `Info2` with basename, when it's not being used?

Reply:
Can this be a future feature request that we prioritize? The main purpose of these PR's is long over due.

Reply:
Yup, a separate PR would be fine.